src/gui/MiniMap.cpp

```diff
@@ -333,11 +333,11 @@ void MiniMap::showBookMiniMap(int showLevel) {
 		
 		if(showLevel == ARX_LEVELS_GetRealNum(m_currentLevel)) {
 			playerPos = computePlayerPos(zoom, showLevel);
-			start = Vec2f(490.f, 220.f) - playerPos;
+			start = Vec2f(rect.center()) - playerPos;
```

Comment:
If I recall correctly, this was just a little bit off the center so I assumed that someone forgot to update the value or thought it looks better that way (hardly noticeable IMHO). The center of the rectangle is much more easily scalable.